### PR TITLE
Changed code into JS

### LINKED ISSUE #1284: Replace code examples in sorting alogrithms with JS
Currently [sorting.md](https://github.com/Techtonica/curriculum/blob/master/algorithms/sorting.md) uses Python code examples. We want to replace this with JavaScript.

- In addition there is a style typo near BogoSort

### PR DESCRIPTION
**Pull request by goyalmuskan**
Fixed #1284

### CODE REVIEW

**Comment by goyalmuskan** [comment]
@Kundan28 @vegetabill

File: algorithms/sorting.md [hunk 1]
```diff
@@ -94,11 +94,11 @@ Let's break into groups again and try out the sorting algorithm on our own decks
 What's the Complexity of Bubble Sort?
 
 ```JavaScript
-function bubble_Sort(L)
+function bubble_Sort(a)
 {
     var swap;
     var n = L.length-1;
```
Comment (Kundan28) [inline algorithms/sorting.md]: Make it `var n = a.length-1;`

**Comment by goyalmuskan** [comment]
Thank you @Kundan28, I have made the changes.

**Comment by Kundan28** [comment]
> I have one request: could you please change all the `var`s to const or let?
> 
> Also, is `do` JavaScript? I am not familiar with it.

But there's other issue for that.

**Comment by goyalmuskan** [comment]
I have made the necessary changes @alodahl @Kundan28

**Comment by alodahl** [comment]
@Kundan28 the issue you mentioned does not ask them to convert `def` to javascript, so we need the change here.  Also, @goyalmuskan please don't use `do`.

**Comment by Kundan28** [comment]
> @Kundan28 the issue you mentioned does not ask them to convert `def` to javascript, so we need the change here. Plus, that issue is almost ready to merge.

Okay, my apology for the mistake.

**Comment by goyalmuskan** [comment]
@alodahl @Kundan28 I have made the changes, please check.

File: algorithms/sorting.md [hunk 2]
```diff
@@ -59,9 +59,8 @@ Say we want to sort a deck of cards:
 Let's talk about the Complexity of BogoSort. Here's the pseudocode:
 
 ```JavaScript
-while not Sorted(a) do
+while not Sorted(a) 
    Shuffle(a)
```
Comment (Kundan28) [inline algorithms/sorting.md]:
```suggestion
while not Sorted(a):
    Shuffle(a)
```

**Comment by Kundan28** [comment]
Update in scoresheet.

**Pull request merged**